ocean_provider/validation/requests.py

```diff
@@ -21,15 +28,36 @@ class CustomJsonRequest(JsonRequest):
     def __init__(self, request=None):
         request = request or flask_request
         request = get_request_data(request)
-        self._validator = CustomValidator(
-            rules=self.rules(),
-            messages={
-                "signature.signature": "Invalid signature provided.",
-                "signature.download_signature": "Invalid signature provided.",
-            },
-            request=request,
+        class_name = self.__class__.__name__
+        self._validators = list()
+        if os.getenv("RBAC_SERVER_URL") and class_name in [
+            "InitializeRequest",
```

Comment:
i.e. here we can reuse the action mapper keys instead of rewriting this array by hand